src/librbd/CMakeLists.txt

```diff
@@ -300,6 +300,8 @@ if(WITH_RBD_RWL OR WITH_RBD_SSD_CACHE)
     blk
     ceph-common
     cls_rbd_client
+    libneorados
+    librados
```

Comment:
has there been any issue around this? But librbd is already including these?[0]

[0]https://github.com/ceph/ceph/pull/39015/files#diff-6fa991d403fabb30b9736fe4e3e6091b98b4719fdbe8a1f6c8592ac51eadccc2L325-L330

Reply:
This pwl_cache library needs to depend on these two libraries as it uses related interfaces.   
Or else fio doesn't work with writeback-enabled images.

Reply:
Do you see a specific error? When I run the following, it seems to load and the plugin just fine (the cache gets deadlocked after a couple IOs):

```
rbd --rbd_plugins=pwl_cache --rbd_persistent_cache_mode=ssd bench-write image1
```

Reply:
Yes, when running with fio, it shows the error like 'undefined symbols read_snap()'. 
I guess the command `rbd` can work because it links the library `libneorados` in its CMakeLists.txt.

Reply:
Ah, so it's probably really just an issue w/ `libneorados` right since it's statically linked.